egs/wsj/s5/utils/data/perturb_data_dir_speed_3way.sh

```diff
@@ -53,8 +59,8 @@ echo "... would be very slow, and you might need them."
 utils/data/get_utt2dur.sh ${srcdir}
 utils/data/get_reco2dur.sh ${srcdir}
 
-utils/data/perturb_data_dir_speed.sh 0.9 ${srcdir} ${destdir}_speed0.9 || exit 1
-utils/data/perturb_data_dir_speed.sh 1.1 ${srcdir} ${destdir}_speed1.1 || exit 1
+utils/data/perturb_data_dir_speed.sh --include-spk-prefix $include_spk_prefix 0.9 ${srcdir} ${destdir}_speed0.9 || exit 1
+utils/data/perturb_data_dir_speed.sh --include-spk-prefix $include_spk_prefix 1.1 ${srcdir} ${destdir}_speed1.1 || exit 1
```

Comment:
this may be problematic in terms of data sorting.  generally speaking it's safest to always ensure that speaker ids are prefixes of utterance-ids .  It would be better to make the utterance affix a postfix instead of a prefix, in the case where no speaker prefix is used.

Reply:
@danpovey yes this morning I found such issue with one of the SRE datasets. Will handle it.